traitsui/examples/demo/Standard_Editors/DatetimeEditor_demo.py

```diff
@@ -34,7 +34,7 @@
 class DateEditorDemo(HasTraits):
```

Comment:
Off-topic for this PR, but the `DateEditorDemo` name is surprising, giving that we're editing a `Datetime` trait rather than a `Date` trait.

Reply:
Likely either copy/paste error or reflecting the fact that there was no Datetime trait until comparatively recently.